services/generic/color_control.md

```diff
@@ -1,45 +1,104 @@
-### Color control service
+# Color Control Service
 
-The service has to be used to control color components of a lightning device.
+The service can be used to control color components of a lighting device, such as LED strips or bulbs.
 
-#### Service names
+## Service name
 
 `color_ctrl`
 
-#### Interfaces
+## Interfaces
 
-Type | Interface                  | Value type |  Description
------|----------------------------|------------|-------------------
-in   | cmd.color.get_report       | null       | The command is a request for a map of color component values
-in   | cmd.color.set              | int_map    | value is a map of color components. val= {"red":200, "green":100, "blue":45}
-out  | evt.color.report           | int_map    | Map of color components, where value is component intensity.
--|||
-in   | cmd.color.start_transition | object     | Example value: val= {"component": "red", "transition": "up", "duration": 127}
-in   | cmd.color.stop_transition  | str        | Stop fading/enhancing single color component. val = "red"
+| Type | Interface                  | Value type | Description                                                                                                                                                      |
+|------|----------------------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| in   | cmd.color.get_report       | null       | Requests a map of color component values.                                                                                                                        |
+| in   | cmd.color.set              | int_map    | Sets values for all color components using a map in which key is a [`color_component`](#definitions) listed in [`sup_components`](#service-properties) property. |
+| out  | evt.color.report           | int_map    | Returns map of color components, where value is the component intensity.                                                                                         |
+| -    |                            |            |                                                                                                                                                                  |
+| in   | cmd.color.start_transition | object     | Starts a single color component transition, see the [`transition`](#definitions) object definition for more details.                                             |
+| in   | cmd.color.stop_transition  | str        | Stop fading/enhancing single color component. val = "red"                                                                                                        |
```

Comment:
Stop changing brightness of a single color. <- maybe this way ?

Reply:
It's not brigthness, it's value of the color component. In case of RGB it's the value of R, G or B, in case of cold_w, warm_w, temp it's not brithness but temperature. It's brithness/lightness only in the case of HSL, which we do not support fully.

Reply:
Rewritten to "Stops transition of the single color component provided in the value.`

Good catch, this one slipped.